analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocations.cs

```diff
@@ -150,28 +213,30 @@ public class Sample
     static object staticField1 = null;
     static object staticField2 = null;
 
-    void CallToMethodsShouldResetFieldConstraints()
+    void OtherInvocationsShouldNotResetFieldConstraints()
```

Comment:
The purpose is in the test name
```suggestion
    void Main()
```